src/Components/Input.jsx

```diff
@@ -0,0 +1,33 @@
+import PropTypes from "prop-types";
+export function Input(props) {
+  const { inputValue, handleInputChange } = props;
+  return (
+    <input
+      type="text"
+      className="border border-slate-300  rounded-md pl-2 h-8 hover:shadow-lg"
+      placeholder="Your Todo..."
+      value={inputValue}
+      onChange={handleInputChange}
+    />
+  );
+}
+Input.propTypes = {
+  inputValue: PropTypes.string.isRequired,
+  handleInputChange: PropTypes.func.isRequired,
+};
+export function InputCheckbox({ checked, onChange }) {
```

Comment:
dont create another input for checkbox 
use the above input itself for checkbox

Reply:
why this is not resolved yet ?

Reply:
ma'am checkbox and input 
don't have any common props that's why i use different input

Reply:
use only one common input for both todo input and checkbox
fix this